Tests/ClientTests/TabLocationViewTests.swift

```diff
@@ -0,0 +1,42 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+@testable import Client
+
+class TabLocationViewTests: XCTestCase {
+    func testDelegateMemoryLeak() {
+        let tabLocationView = TabLocationView()
+        let delegate = MockTabLocationViewDelegate()
+        tabLocationView.delegate = delegate
+        delegate.cleanup()
+        trackForMemoryLeaks(delegate)
```

Comment:
Just adding this line here then all good!
```suggestion
        trackForMemoryLeaks(tabLocationView)
        trackForMemoryLeaks(delegate)
```